spark/src/test/scala/org/apache/spark/sql/delta/rowid/RowIdCloneSuite.scala

```diff
@@ -70,6 +71,22 @@ class RowIdCloneSuite
     }
   }
 
+  test("self-clone an empty table") {
+    for (enableRowIdsForSource <- BOOLEAN_DOMAIN) {
+      withTables(
+        TableSetupInfo(tableName = "source",
+          rowIdsEnabled = enableRowIdsForSource, tableState = TableState.EMPTY)) {
+        cloneTable(targetTableName = "source", sourceTableName = "source")
+
+        val (targetLog, snapshot) =
+          DeltaLog.forTableWithSnapshot(spark, TableIdentifier("source"))
```

Comment:
nit: put it on the same line
```suggestion
        val (targetLog, snapshot) = DeltaLog.forTableWithSnapshot(spark, TableIdentifier("source"))
```